### DISCUSSION TITLE
The Group component doesn't match the size of its parent svg tag

### DISCUSSION THREAD

**Discussion by JoeVanGundy** [airbnb/visx#1322] [Q&A]
Hi there!

I'm trying to make a `<LinePath/>` fit perfectly inside of a box.
No matter what I try, it seems that the Group component always has a smaller size that it should.
Meaning I need to arbitrarily add margins to the plot which causes some jank when I try to reuse the plots.

Is there something I need to do to always make sure the group takes the size of its parent `<svg/>` tag?
Or is there something wrong with responsiveness of the components?

What the error looks like. The group should be fitting inside of the black border.
My expectation would be that it's size is 219px by 60px from the params derived from the ParentSize compnent.
'
<img width="423" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/8807246/130163597-7e3277b6-b6f2-428e-b438-b943fbb56350.png">

I'm getting the parent container size using `<ParentSize/>`:
![image](https://user-images.githubusercontent.com/8807246/130163089-1d0f5ef3-e074-4a81-b642-d5d21fb377bb.png)

And my scales look like this:
![image](https://user-images.githubusercontent.com/8807246/130162837-2dc3acaa-58af-47e3-880e-41921de6b615.png)

My plot looks like this:
![image](https://user-images.githubusercontent.com/8807246/130162797-b1c5db92-b948-4f7d-afe4-bbd772bce4b2.png)

**Comment by williaster** [comment]
Hey @JoeVanGundy 👋  I'm going to convert this to a discussion because I don't think it's an issue. 

It could be useful to share a sandbox to tinker more, but overall I don't thing `Group` is doing anything here. It is simply a [`<g />`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g) element, which just expands to its child dimensions. Often you would use a `<g />` to apply a transform to a group of elements (e.g,. to apply a chart `left/top` margin), but since you don't supply any transform its dimensions are only coming from the line path ... so then the question is why is the line path not "full width/height"? 😅 

Again, a sandbox would be useful, but I think your `path` may not take the entire available `width/height` because of some of your scale settings:
- by definition, [`nice`](https://github.com/d3/d3-scale#continuous_nice) will expand your scale so that it starts/stops on even values (this is mostly useful for axis ticks). that means if your `data` goes from `2-8`, the available space will potentially be used to represent `0-10`, meaning your line doesn't take the full space because it starts after `0` and ends before `10`. I often will render an axis even if I ultimately remove it to ensure the data is doing what I want 😄 
- you're setting a max time domain, which may or may not match the data range. ^for the same reason as `nice`, this might compress the line path into a smaller space than is available.

Hope that helps, I think to debug more I would need a sandbox to tinker with.
